src/xpublish_tiles/grids.py

```diff
@@ -287,7 +287,7 @@ def __init__(
             # the line at 180, -180; calculate the neighbours of those faces; extract those vertices and pad those.
             # However this does not work if the boundary faces without padding don't intersect those bounding longitudes
             # e.g. consider if vertices are between -178.4 and +178.4, how do we connect the boundary without the convex hull approach?
-            boundary = np.unique(triangle.convex_hull(vertices))
+            boundary = pd.unique(triangle.convex_hull(vertices).ravel())
```

Comment:
would be unnecessary with latest numpy's new hash-bsaed unique but we can't do that yet because of numba. Use pandas' hash-based unique instead